src/cli/main.c

```diff
@@ -66,6 +67,11 @@ main(int argc, char *argv[])
 			del_network = 1;
 			network_name = optarg;
 			break;
+		case 'v':
+			#ifdef VERSION 
```

Comment:
You can check here https://cmake.org/cmake-tutorial/, grep for "Adding a Version Number and Configured Header File". They show how cmake can create a version.h with the version defined there, that we just need to include

Reply:
ahhh gotcha gotcha. will take a looksie!